### PR TITLE
footer added

### PR DESCRIPTION
**Pull request by Bisalkumar1**
footer added

### CODE REVIEW

**Comment by Vijaykv5** [comment]
What is this PR? we need our footer

**Comment by Bisalkumar1** [comment]
Footer added

**Comment by Vijaykv5** [comment]
Our Website name is Snap Grid, Not ph

**Comment by Vijaykv5** [comment]
> > Our Website name is Snap Grid, Not ph
> 
> What was ph in image which u provide me to create as same

I said you to take inspiration of website mate..
not to create the exact same one

**Comment by Vijaykv5** [comment]
Also Please make the footer responsive too

**Comment by Bisalkumar1** [comment]
updated and it is already responsive

**Comment by Vijaykv5** [comment]
it's not **responsive** 
Screenshot 2023-10-27 at 5 45 34 PM

Also add the Links also like for eg in github, MIT licence and documentation.

Also remove discord , linkedin and github icons as we don't have that socials. Add Github link to **"star this project"** button

**Comment by Bisalkumar1** [comment]
chages done for responsiveness and add github link to button

File: src/Pages/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+   <>
+    <footer>
+        <div className="mainfooter">
+            <div className="firstfoter">
+                <p className='footerbold'>Snap Grid</p>
+                <p>Projectshut is an open source web app that enables users to freely
```
Comment (Vijaykv5) [inline src/Pages/Footer.jsx:10]: Change this to Snapgrid

Comment (Vijaykv5) [reply src/Pages/Footer.jsx:10]: Also replace that Sentance with : 'SnapGrid serves as a multifaceted online tool that streamlines the process of finding and obtaining images.'

**Comment by Vijaykv5** [comment]
The responsiveness is not correctly fixed @Bisalkumar1 ..
Please check that too -

Screenshot 2023-10-29 at 1 00 47 AM

**Comment by Bisalkumar1** [comment]
done . and for some other screen width space will be remain like that , but tried to remove and did it  .

**Comment by Vijaykv5** [comment]
I'm closing this PR as now as someone thad already done the Result I expected and your PR need more improvements

**Pull request closed without merging**